e2e/fixtures/register.test.js

```diff
@@ -0,0 +1,22 @@
+import { URLS } from '../utils/route.util';
+import { RegisterElements } from '../page-objects/register.po';
+//Declare chance
+const Chance = require('chance');
+const chanceObj = new Chance();
+
+const User = chanceObj.name();
+const Username = User.toLowerCase();
+const Email = Username.replace(' ', '-') + '@gmail.com';
+const passwords = "Testing";
```

Comment:
this looks good, but plz move this util code into a util function.
TODO:
- create a new file: person-util.js
- create a function that returns this object: `{username: 'a use name', password: 'a-use-name@gmail.com'}`
- use this file in register.test.js file